snd/test/src/org/labkey/test/tests/snd/SNDTest.java

```diff
@@ -2190,13 +2200,96 @@ public void categoryPermissionsUI() throws Exception
 
         for (int j = 0; j < categoryRows.size(); j++)
         {
-            value = getPerimissionTableValue(categoryRows.get(j), 1);
+            value = getPermissionTableValue(categoryRows.get(j), 1);
             assertNotNull(value);
             assertTrue(value.equals(permissions.get(j)));
         }
 
     }
 
+    @Test
+    public void testQueryProvisionedSnapshot()
+    {
+        String SOURCE_QUERY_NAME = "SND: Source query";
+        String SNAPSHOT_NAME = SOURCE_QUERY_NAME + " Snapshot";
+        String SOURCE_QUERY = "SELECT\n" +
```

Comment:
All-caps naming is typically used for static class constants (e.g. `private static final String PROJECTNAME`), not variables within a function.